Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataDocumentFactory.cs

```diff
@@ -60,6 +60,10 @@ public string Create(IOpenApiMetadataResourceStrategy resourceStrategy, OpenApiM
                     },
                     host = "%HOST%",
                     basePath = "%BASE_PATH%",
+                    schemes = 
+                        new List<string> {
+                            "%SCHEME%"
+                        },
```

Comment:
lets be consistent to the publishing json, and use an array instead of a list.

Reply:
updated to string array